### ISSUE TITLE
Blank pages in symbol portfolio

### ISSUE THREAD

**Issue by Aurifier** [XCompWiz/Mystcraft-Issues#63]
Hiya! I put some blank pages into a symbol portfolio and now I can't take them out. While I realize this was a dumb idea in the first place, I imagine this isn't intended behavior. Let me know if any details or other help is needed!

**Comment by XCompWiz** [comment]
Blank pages?  As in paper, or pages which lack symbols?
And that's certainly not intended. I can check it out. :)

**Comment by Aurifier** [comment]
Pages which lack symbols. Thanks so much for your awesome work, by the way. Mystcraft is one of those mods I just won't bother playing modded at all without. :D

**Comment by XCompWiz** [comment]
OK, I've solved this (and something else that wasn't completely related).
I started by trying to create a blank page by using paper. Paper can only be put into folders, not portfolios, so I tried putting paper into a folder and then dumping the folder into the portfolio.
The paper vanished. At first I thought it wasn't visible, but then i realized it had been lost.
Fixed that, not it doesn't get copied over.
So that was the semi-related bug. :P

Then I used NEI to spawn a page without any symbol data. The thing about these pages is they are broken; their NBT data isn't ever created.  I expect something along these lines is where yours came from.
I then put them into the portfolio. Fine.
They didn't come out, as you had said.  Turns out the problem was matching the target of the pickup to the original item. Since the items saved to the portfolio didn't have NBT, and the items coming out of the portfolio were quickly given NBT, they stopped matching.  I fixed this by making it so that "getting" information from the blank pages wouldn't alter their state, so they continue to match. THe pages can come out now.

Thanks for finding this and I'm really glad you like the mod. :)
